shared/js/content-scope/fingerprint.js

```diff
@@ -145,3 +145,93 @@ function initCanvasProtection (args) {
         HTMLCanvasElement.prototype[methodName] = methodProxy
     }
 }
+
+// eslint-disable-next-line no-unused-vars
+function initAudioProtection (args) {
+    const { sessionKey, stringExemptionList, site } = args
+    initExemptionList(stringExemptionList)
+    const domainKey = site.domain
+
+    // In place modify array data to remove fingerprinting
+    function transformArrayData (channelData) {
+        const cdSum = channelData.reduce((sum, v) => {
+            return sum + v
+        }, 0)
+        const audioKey = getDataKeySync(sessionKey, domainKey, cdSum)
+        let pixel = audioKey.charCodeAt(0)
+        for (const i in audioKey) {
+            let byte = audioKey.charCodeAt(i)
+            for (let j = 8; j >= 0; j--) {
+                const pixelCanvasIndex = pixel % channelData.length
```

Comment:
Is this loop close enough to the canvas algo that we could combine into a function? We should probably rename this var to audio if you keep it as is..